.gitignore

```diff
@@ -0,0 +1,4 @@
+.DS_Store
```

Comment:
All of these should simple not be committed, you can add `.DS_Store` to your global git config and the rest have no legitimate reason to be included.

Reply:
I disagree. Why doesn't we have .gitignore? It's actually common.

Reply:
It's fine with me - .gitignore is cheap

Reply:
It maybe cheap, but why don't we have these developer unique ignores in Deno then?

Reply:
I'm fine with that in Deno too *shrug*